### PR TITLE
Toggle privacy features on

### PR DESCRIPTION
**Pull request by jaril**
This patch:
1) Toggles the privacy features on 🎉
2) Makes it so that only the recording owner can set the recording privacy (private/public) and edit collaborators

### CODE REVIEW

File: src/ui/utils/prefs.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ pref("devtools.toolbox-opened", true);
 pref("devtools.features.comments", true);
 pref("devtools.features.users", true);
 pref("devtools.features.auth0", true);
-pref("devtools.features.private", false);
+pref("devtools.features.private", true);
```
Comment (jasonLaster) [inline src/ui/utils/prefs.js:23]: can we get rid of this pref :)

File: src/ui/components/DevTools.js [hunk 2]
```diff
@@ -65,15 +67,8 @@ function getUploadingMessage(uploading) {
...
-  if (!features.private) {
-    return true;
-  }
-
   // We let Hasura decide whether or not the user can view a recording. The response to our query
   // will have a recording if they're authorized to view the recording, and will be empty if not.
-  // What this doesn't explicitly tell us is *why* that user is allowed to view the recording: is the
-  // recording public, or is the user the original creator for that recording? You can check that
-  // by checking the recording's user field, which will be null for non-creator users.
   return data.recordings.length > 0 ? true : false;
```
Comment (jasonLaster) [inline src/ui/components/DevTools.js:72]: data can be undefined, so i recommend doing

```
return data?.recordings.length > 0
```

the ternary isn't needed 😉

**Review by jasonLaster — APPROVED** [review]
Looks great. Ship it!

**Comment by bhackett1024** [comment]
Hi, sorry, but I had to revert this change because it broke all of our tests both when running locally and as a github action.

**Pull request merged**